doc/WebSite/Dapper-Usage.html

```diff
@@ -0,0 +1,187 @@
+{
+    public override void Initialize()
+    {
+        IocManager.RegisterAssemblyByConvention(typeof(SampleApplicationModule).GetAssembly());
+    }
+}
+			</pre>
+			
+			<h4>When entity classes and tables have different names</h4>
+
+			<p>For example, <strong>User</strong> class reference <strong>AbpUsers</strong> table. So you should define a mapping between them.</p>
+			
+			<pre lang="cs">public class UserMapper : ClassMapper&lt;User&gt;
+{
+    public UserMapper()
+    {
+        //use different table name
+        Table("AbpUsers");
+        
+        Map(x => x.Logins).Ignore();
+        Map(x => x.Roles).Ignore();
+        Map(x => x.Claims).Ignore();
+        Map(x => x.Permissions).Ignore();
+        Map(x => x.Settings).Ignore();
+
+        Map(x => x.DeleterUser).Ignore();
+        Map(x => x.CreatorUser).Ignore();
+        Map(x => x.LastModifierUser).Ignore();
+        Map(x => x.FullName).Ignore();
+        Map(x => x.Tokens).Ignore();
+
+        AutoMap();
+    }
+}</pre>
+			
+			<p>After define mappings, you need to register them in module initialize method.</p>
+						<pre lang="cs">[DependsOn(
+     typeof(AbpEntityFrameworkModule),
+     typeof(AbpDapperModule)
+)]
+public class SampleApplicationModule : AbpModule
+{
+    public override void Initialize()
+    {
+        IocManager.RegisterAssemblyByConvention(typeof(SampleApplicationModule).GetAssembly());
+        <strong>DapperExtensions.DapperExtensions.SetMappingAssemblies(new List<Assembly> { typeof(SampleApplicationModule).GetAssembly() });</strong>
+    }
+}
+			</pre>
+			
+			<h3 id="DocUsage">Usage</h3>
+
+			
+			<p>After registering <strong>AbpDapperModule</strong>, you can use it like traditinal ABP repositories like following.</p>
+			<pre lang="cs">public class SomeDomainService : ITransientDependency
+{
+    private readonly IDapperRepository&lt;Animal&gt; _animalDapperRepository;
+    private readonly IRepository&lt;Animal&gt; _animalRepository;
+    private readonly IDapperRepository&lt;Person&gt; _personDapperRepository;
+    private readonly IRepository&lt;Person&gt; _personRepository;
+    private readonly IUnitOfWorkManager _unitOfWorkManager;
+
+    public SomeDomainService(
+        IUnitOfWorkManager unitOfWorkManager,
+        IRepository&lt;Person&gt; personRepository,
+        IRepository&lt;Animal&gt; animalRepository,
+        IDapperRepository&lt;Person&gt; personDapperRepository,
+        IDapperRepository&lt;Animal&gt; animalDapperRepository)
+    {
+        _unitOfWorkManager = unitOfWorkManager;
+        _personRepository = personRepository;
+        _animalRepository = animalRepository;
+        _personDapperRepository = personDapperRepository;
+        _animalDapperRepository = animalDapperRepository;
+    }
+
+    public void DoSomeStuff()
+    {
+        using (IUnitOfWorkCompleteHandle uow = _unitOfWorkManager.Begin())
+        {
+            _personRepository.Insert(new Person("John"));
```

Comment:
Keep sample codes minimal. All examples you have written already valid for regular repositories and no point to write them all. Why not writing a simple "Select * From ..." example? Dapper's main advantage is to use direct SQL and map to entity fastly.

Also, PersonRepository is enough, remove animal repository since it does not add a value.